Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -40,21 +40,9 @@ - (instancetype)initWithFrame:(CGRect)frame
     return self;
 }
 
-- (instancetype)init
-{
```

Comment:
`[UIView init]` invokes `[self initWithFrame:...]`.  I don't like to bake that assumption into the code since a lot of people don't know that.  Anyway, I guess I'll leave things as they stand but add comments.